### ISSUE TITLE
R script executor: Heatmap script can't be executed

### ISSUE THREAD

**Issue by dennishendriksen** [molgenis/molgenis#8384]
### How to Reproduce
See test case SCRIPTSYS004
1. Create a 'data'-group
2. Import the [Rtest.xlsx](https://github.com/molgenis/molgenis/files/3053928/Rtest.xlsx) script in the 'data'-folder from the data group
3. "Create script Heatmap

Name: Heatmap
Type: R
Content: 

  library(RCurl)
  eval(expr = parse(text = getURL(""https://latest.test.molgenis.org/molgenis.R?molgenis-token=${molgenisToken}"")))
  df <- molgenis.get(""data_Rtest"")
  df2 <- apply(df, 1,as.numeric)
  is.na(df2) <- sapply(df2, is.infinite)
  df2[is.na(df2)] <- 0
  df2[is.nan(df2)] <- 0
  png('${outputFile}')
  heatmap(as.matrix(df2),symm=FALSE,trace=""none"")
  dev.off()
  molgenis.logout()

Generate security token: Yes
Result file extension: png
"

### Expected behavior
Script runs succesfully

### Observed behavior
1) Test fails
```
10:52:25.029 - Execution started.
10:52:25.955 - Failed. no lines available in input

In call:
read.table(file = file, header = header, sep = sep, quote = quote, 
    dec = dec, fill = fill, comment.char = comment.char, ...)
```
2) Timestamps incorrect

**Comment by mswertz** [comment]
pls check, in particular to see if this is a general or specific issue

**Comment by fdlk** [comment]
Fixed in 8.0 merge
